my-react-app/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+import LoginForm from './layout.js';
+
+class LoginFormContainer extends Component {
+  submit = values => {
```

Comment:
Don't use `{}` if you have only 1 line of code. Do 
`submit = values => window.alert(JSON.stringify(values, null, 4)); `